jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -21,11 +22,11 @@
                 - culprits
     scm:
         - git:
-            url: https://github.com/kubernetes/kubernetes
+            url: '{url}'
```

Comment:
> Our configs won't work on other Jenkins instances without modification anyway. 

Which basically means any community contributor attempting to extend/fix the Jenkins setup is fighting an uphill battle every step of the way.

That said, changed made